newIDE/app/src/ExportAndShare/ShareDialog/ExportLauncher.js

```diff
@@ -392,9 +412,12 @@ export default class ExportLauncher extends Component<Props, State> {
           {
             gameName: project.getName(),
             gameVersion: project.getVersion(),
-          }
+          },
+          !!payWithCredits
         );
         setStep('build');
+        // Refresh limits as either the quota or the credits may have changed.
+        this.props.authenticatedUser.onRefreshLimits();
```

Comment:
No need to await for this?

Reply:
I don't think we need to, this is just to let the interface update while the build continues to run. I'll leave a comment